platform/ext/target/nordic_nrf/nrf5340/partition/flash_layout.h

```diff
@@ -74,7 +74,7 @@
  * swapping.
  */
 #define FLASH_AREA_BL2_OFFSET      (0x0)
-#define FLASH_AREA_BL2_SIZE        (0x10000) /* 64 KB */
+#define FLASH_AREA_BL2_SIZE        (0xF000) /* 60 KB */
```

Comment:
This looks sufficient - I wonder if we should update upstream Zephyr to use 60kB (0xF00) or even 48kB, which is the case for nRF52